ArticleTemplates/assets/scss/base/_typography.scss

```diff
@@ -145,35 +145,29 @@ $fontMap: (
     // LEGACY FONTS:
 
     // Display Egyptian
-    GuardianEgyptianWeb100: (
```

Comment:
Is there no interactives using this?

Reply:
Yes there might be, but there's no _Thin_ weight for _Guardian Headline_, so it will automatically pick up _Light_ at weight 200